pkg/policy/api/fqdn.go

```diff
@@ -67,9 +67,13 @@ type PortRuleDNS FQDNSelector
 
 // Sanitize checks that the matchName in the portRule can be compiled as a
 // regex. It does not check that a DNS name is a valid DNS name.
-func (kr *PortRuleDNS) Sanitize() error {
-	// All L7 toFQDNs matchNames are regexes
-		return err
+func (r *PortRuleDNS) Sanitize() error {
```

Comment:
Same here re: unit tests. Can be done in a follow-up PR / tracked with a follow-up issue.